IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -269,6 +270,14 @@ - (nullable instancetype)initWithGroup:(nullable NSString *)keychainGroup
                                    
                                    };
 
+        NSString *appIdentifier = [[NSBundle mainBundle] bundleIdentifier];
+        if (!appIdentifier)
```

Comment:
why would there be no bundle identifier?

Reply:
it's not set when running as a unit test from xctest

Reply:
I'm generally cautious for writing code that executes only in unit tests. Can this be implemented differently?

Reply:
I realized the tests work fine using bundleIdentifier, so I eliminated using the test-only fallback of processName.